.github/workflows/ci-sharded.yml

```
@@ -0,0 +1,331 @@
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: comic_pile_test
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        command: postgres -c max_connections=200
```

actionlint (1.7.10)

[error] unexpected key "command" for "services" section. expected one of "credentials", "env", "image", "options", "ports", "volumes"

(syntax-check)